RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  PaywallViewController.swift
+//  
+//
+//  Created by Nacho Soto on 8/1/23.
+//
+
+import RevenueCat
+import SwiftUI
+import UIKit
+
+/// A view controller for displaying `PaywallData` for an `Offering`.
+///
+/// - Seealso: ``PaywallView`` for `SwiftUI`.
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
```

Comment:
Will this need to get dropped to `iOS 15.0` after #2933 gets merged?

Reply:
Yes (**if** it gets merged, we'll see)